cgi/BibRights.pm

```diff
@@ -0,0 +1,72 @@
+package BibRights;
+
+# A CRMS-oriented wrapper around bib_rights.pm using HathiTrust Bib API
+# to retrieve MARC XML.
+
+use strict;
+use warnings;
+use utf8;
+
+use Capture::Tiny;
+use Data::Dumper;
+use MARC::File::XML(BinaryEncoding => 'utf8');
+use MARC::Record;
+
+use lib $ENV{SDRROOT} . '/crms/post_zephir_processing';
+use bib_rights;
+use Metadata;
+
+binmode(STDOUT, ':encoding(UTF-8)');
+
+our @BIB_RIGHTS_INFO_FIELDS = qw(attr bib_fmt bib_key date1 date2 date_desc
+  date_munged date_type date_used desc gov_pub id orig_date1 orig_date2
+  pub_country pub_place reason us_fed_doc vol_date);
+
+
+sub new {
+  my ($class, %args) = @_;
+  my $self = bless {}, $class;
+  return $self;
+}
+
+# Returns a structure {entries => [bib_rights_info+], optional error => errstr}
+sub query {
+  my $self = shift;
+  my $id   = shift;
+
+  my $br;
+  # bib_rights.pm likes to emit cutoff year debugging info to STDERR.
+  # So suppress it.
+  my ($stderr, @result) = Capture::Tiny::capture_stderr sub {
```

Comment:
Yuck. Worth an update to bib_rights.pm?

Reply:
This makes it so I don't *have* to wade into PZP/bib_rights right now... but yes. There are several actions I would like to take in there wrt print to STDERR before long.

Reply:
Let's add a task to the post-zephir processing epic then

Reply:
DEV-456